src/Resources/config/core.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <defaults autowire="true" autoconfigure="true"/>
```

Comment:
You don't need this. Maybe you are missing one tag to the controller:

```
<tag name="controller.service_arguments"/>

Reply:
I already test that before and it’s not working

Reply:
Can you copy the error you get then adding the tag and removing this line?

Reply:
"sonata.admin.controller.crud" has no container set, did you forget to define it as a service subscriber?

Reply:
Could you copy the full stack trace of the error, maybe we can find why it is not working.

Reply:
I had any stack trace, the only feedback i have is this one : 

`[critical] Uncaught PHP Exception LogicException: ""sonata.admin.controller.crud" has no container set, did you forget to define it as a service subscriber?" at /Users/mpetrini/Documents/Projets/perso/sonata-admin-bundle/vendor/symfony/framework-bundle/Controller/ControllerResolver.php line 39
`